### ISSUE TITLE
Monocular camera: scale changes (actually diminished) while moving around 

### ISSUE THREAD

**Issue by joshi-bharat** [uzh-rpg/rpg_svo_example#28]
I am trying to move Clearpath husky around the corridor with loops using monocular camera. But the map scale changes while moving the robot around. Are there any specific parameters I can tweak? I am not sure if adding IMU will help or not. Here are some images for reference...
![husky_2](https://user-images.githubusercontent.com/17511684/35783743-5617aff6-09da-11e8-9c5d-c38d9eedb3ce.png)
![husky_indoor_120_5](https://user-images.githubusercontent.com/17511684/35783744-56ec7ce0-09da-11e8-9e0b-c27159f6bc1d.png)

**Comment by skandaswaroop1998** [comment]
hi did you fix the issue? i am facing the same problem

**Comment by kylelindgren** [comment]
Same ^^ any explanation or possible solutions would be much appreciated.

**Comment by joshi-bharat** [comment]
> hi did you fix the issue? i am facing the same problem

Nope. This is one of the limitations of the package using the monocular camera. I am not sure it can be fixed easily.

**Comment by kylelindgren** [comment]
That's unfortunate. But thank you for the response.
